javascript.js

```diff
@@ -0,0 +1,134 @@
+
+/* elements & variables */
+var navBarProfilePic = document.getElementsByClassName("navBar__bgPicture")[0];
+var navBarDownloadCV = document.getElementsByClassName("navBar__CVbtn")[0];
+var professionalSection = document.getElementById("professional");
+var homeDownloadCV = document.getElementsByClassName("home__downloadCV")[0];
+var SideBar = document.getElementById("navBar");
+var porcentageBars = document.getElementsByClassName("professional__percentBar");
+var verticalLine = document.getElementsByClassName("experince__line")[0];
+var vectorsPoints = document.getElementsByClassName("experince__point");
+var flipButton = document.getElementsByClassName("flip__btn");
+var awardsSection = document.getElementById("awardsInterest");
+var navBarScrolled = false;
+
+/* FUNCTIIONS */
+
+/* General Functions */
+function detectmob() { 
+    if( navigator.userAgent.match(/Android/i)
+    || navigator.userAgent.match(/webOS/i)
+    || navigator.userAgent.match(/iPhone/i)
+    || navigator.userAgent.match(/iPad/i)
+    || navigator.userAgent.match(/iPod/i)
+    || navigator.userAgent.match(/BlackBerry/i)
+    || navigator.userAgent.match(/Windows Phone/i)
+    ){
+        return true;
+    } 
+    else {
+        return false;
+    }
+}
+/* Navigation Functions */
+function navBarSlide() {
+    if (document.documentElement.scrollTop >= (homeDownloadCV.offsetTop + homeDownloadCV.clientHeight) && !navBarScrolled) {
+        navBarScrolled = true;
+        removeNavClasses();
+    }
+    if (document.documentElement.scrollTop < (homeDownloadCV.offsetTop + homeDownloadCV.clientHeight) && navBarScrolled) {
```

Comment:
you can simplify here using a variable

```js
const hasScrolled = document.documentElement.scrollTop >= (homeDownloadCV.offsetTop + homeDownloadCV.clientHeight)
```

then if statements going to be

```js
if (hasScrolled && !navBarScrolled) {
        navBarScrolled = true;
        removeNavClasses();
}

if (!hasScrolled && navBarScrolled)
```

Because the opposite of `>=` is `<`.

Note: When conditions are too long we tend to declare them as a variable explaining the intention or as a function with a name that explain the intention.

Reply:
Didn't cross my mind, will do so, ty